src/components/biblio/BiblioEntity.js

```diff
@@ -257,29 +344,116 @@ const EntityCreate = () => {
     return updateJson;
   }
 
+  function checkTopicEntitySetQualifierForSGD() {
+
+    /*
+     -------------------------------------------
+     qualifier = 'primary display', ATP:0000147
+     -------------------------------------------  
+    */
+    const isPrimaryTopic = sgdPrimaryTopics.includes(topicSelect);  
+    const isEntityEmpty = !entityText || entityText.trim() === '';
+    let isEntityInvalid = false;
+    if ( entityResultList && entityResultList.length > 0 ) {
+      for (let entityResult of entityResultList.values()) {
+        if (entityResult.curie === 'no Alliance curie') {
+          isEntityInvalid = true;
+	  break;
+        }
+      }
+    }
+    if (isPrimaryTopic) {
+      if (isEntityEmpty || isEntityInvalid) {
```

Comment:
Do they want that error when any entity is invalid (meaning, everything they enter must be right), or when no entities are valid (meaning at least one entity must be right) ?  I can see the use-case for curators copy-pasting a bunch of stuff and not wanting to find all the things that are wrong and manually delete them.

Reply:
Made a question to curators in the ticket